editoast/src/modelsv2/infra/object_queryable.rs

```diff
@@ -9,21 +9,27 @@ use diesel_async::RunQueryDsl;
 use editoast_schemas::primitives::ObjectType;
 use serde::Deserialize;
 use serde::Serialize;
-use serde_json::Value as JsonValue;
 
 use super::Infra;
 use crate::error::Result;
 use crate::modelsv2::get_geometry_layer_table;
 use crate::modelsv2::get_table;
 use crate::modelsv2::DbConnection;
 
-#[derive(QueryableByName, Debug, Clone, Serialize, Deserialize, PartialEq)]
+editoast_common::schemas! {
+    ObjectQueryable,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, QueryableByName, utoipa::ToSchema)]
+#[schema(as = InfraObjectWithGeometry)]
 pub struct ObjectQueryable {
     #[diesel(sql_type = Text)]
     pub obj_id: String,
     #[diesel(sql_type = Jsonb)]
-    pub railjson: JsonValue,
+    #[schema(value_type = Object)]
```

Comment:
Why not tag it with `InfraObject` ?

Reply:
The problem is that `InfraObject` serializes as `{ obj_type: X, railjson: Y }`: 

```rust
#[derive(Debug, Clone, PartialEq, serde::Deserialize, serde::Serialize, utoipa::ToSchema)]
#[serde(tag = "obj_type", deny_unknown_fields)]
pub enum InfraObject {
    TrackSection { railjson: TrackSection },
```

I don't believe we use an enum just OR-ing all schemas. We could define one for that one use case, but I'm fine leaving it like this, wdyt?

Reply:
Okay, leaving `Object` seems most cost-effective 😁 
I think the real solution would be to update the front so tagging with the current enum would work